src/iuse.cpp

```diff
@@ -3802,9 +3801,7 @@ std::optional<int> iuse::tazer2( Character *p, item *it, const tripoint_bub_ms &
 
 std::optional<int> iuse::shocktonfa_off( Character *p, item *it, const tripoint_bub_ms &pos )
 {
-    if( !p ) {
-        debugmsg( "%s called action shocktonfa_off that requires character but no character is present",
-                  it->typeId().str() );
+    if( !check_character_for_action( p, it, __func__ ) ) {
     }
```

Comment:
```suggestion
    check_character_for_action( p, it, __func__ );
```
Idk why clang is ok with this

Reply:
It shouldn't keep going. This seems like a pre-existing bug that I should fix while I'm here. Thanks for catching it.